cmd/grype/cli/options/database.go

```diff
@@ -18,6 +18,7 @@ type Database struct {
 	AutoUpdate             bool          `yaml:"auto-update" json:"auto-update" mapstructure:"auto-update"`
 	ValidateByHashOnStart  bool          `yaml:"validate-by-hash-on-start" json:"validate-by-hash-on-start" mapstructure:"validate-by-hash-on-start"`
 	ValidateAge            bool          `yaml:"validate-age" json:"validate-age" mapstructure:"validate-age"`
+	MinAgeToCheckForUpdate time.Duration `yaml:"min-age-to-check-for-update" json:"min-age-to-check-for-update" mapstructure:"min-age-to-check-for-update"`
```

Comment:
naming suggestion: what about `stale-age` with an attached description?

Reply:
``` yaml
db:
   min-age-to-check-for-update: 12h
```

vs

``` yaml
db:
   stale-age: 12h
```

The first one seems clearer to me in terms of what it controls. But I agree the name is pretty wordy. How do we communicate to a user what `stale-age` does?

Reply:
I'm open to other naming suggestions too. Here's how we attach descriptions to fields https://github.com/anchore/syft/blob/9573f557d12756f8c873b98a6af476835f611055/cmd/syft/internal/options/update_check.go#L15-L17

Reply:
I was trying to be parallel to `MaxAllowedBuiltAge`, itself perhaps not an ideal age, since "built" seems a bit of an implementation concern.

Reply:
What about `min-age-to-update` ?

Reply:
What about something like `update-check-interval`?